tests/test_openjdk.py

```diff
@@ -156,3 +173,48 @@ def test_jdk_extended(
 
     for check in params.expected_err_strings:
         assert check in testout.stderr
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    CONTAINER_IMAGES_CASSANDRA,
+    indirect=["container_per_test"],
+)
+def test_jdk_cassandra(
+    container_per_test,
+    host,
+    container_runtime: OciRuntimeBase,
+):
+    """TODO info
+    """
+
+    container_per_test.connection.run_expect(
+        [0], "curl -OL https://downloads.apache.org/cassandra/4.0.6/apache-cassandra-4.0.6-bin.tar.gz"
```

Comment:
Is it a good idea to hardcode the version here?

Reply:
I did in a different way